packages/swamp/example/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:18
+
+WORKDIR /app
+
+ENV PORT=80
+
+COPY package*.json ./
+
+RUN npm install && npm install typescript -g
+
+COPY src ./src
+COPY tsconfig.json ./
+
+# this must go after install or else dev deps will be ignored
+ENV NODE_ENV=production
+
+RUN npm run build
+
+# Expose the port the app runs on
+EXPOSE 80
```

Comment:
80?

Reply:
This just has to match `ENV PORT=...` above; can add comment